admin/python/lsst/qserv/admin/dataLoader.py

```diff
@@ -621,8 +629,14 @@ def _createDummyChunk(self, database, table):
             self._log.info('Make dummy chunk table for %r', name)
 
             # just make regular chunk with special ID, do not load any data
-            wmgr.createChunk(database, table, 1234567890, overlap=self.partOptions.isSubChunked)
-
+            try:
+                wmgr.createChunk(database, table, 1234567890, overlap=self.partOptions.isSubChunked)
+            except ServerError as exc:
+                if exc.code == 409:
+                    self._log.info('Dummy chunk 1234567890 exists for table %r', name)
```

Comment:
I think `name` is not the name of the table here but a name of the worker, the message may be confusing if you mix "table" and worker name. Same for the message below.